components/forms/Login/index.jsx

```diff
@@ -36,26 +33,16 @@ const Login = () => {
         >
           Password
         </label>
-        <input
-          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 pr-[23%] focus:outline-none focus:shadow-none"
-          id="password"
-          data-test-id={loginFormTestIds.passwordField}
-          type="password"
-          required
-        />
+        <Input id="password" type="password" dataTestId={loginFormTestIds.passwordField} required={true} className="pr-[23%]" />
```

Comment:
Same as the above comment 🙌🏼 
```suggestion
        <Input id="password" type="password" dataTestId={loginFormTestIds.passwordField} required className="pr-[23%]" />
```

Reply:
done in d129258d877c1e23ba31313cc0390ad9627eb192